teku/src/integration-test/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommandTest.java

```diff
@@ -84,22 +84,31 @@ public void setup(final ClientAndServer server) {
         getMockBeaconServerEndpoint(failoverMockBeaconServer);
 
     argsNetworkOptDefault =
-        new String[] {"vc", "--beacon-node-api-endpoint", mockBeaconServerEndpoint};
+        new String[] {
+          "vc", "--validator-keys=keys:pass", "--beacon-node-api-endpoint", mockBeaconServerEndpoint
```

Comment:
are we implying here that we need to pass `--validator-keys` now?

Reply:
my concern is this implies that we used to have a default location or something, and that will no longer work potentially?

Reply:
Yes, the `vc` subcommand now requires at least one key source as you summarised it in your first comment (file path, external signer url or enabling the key manager API). I chose the `--validator-keys` for the convenience... It could have been `--validator-api-enabled` or `--validators-external-signer-url=http://localhost`.
This doesn't mean we used to have a default location or anything, the tests are simply parsing the command and we weren't requiring any validator keys source at that level

Reply:
the implied default is gone then? it used to be that one of the local keys was a default... im not sure we really want to force a breaking change...

Reply:
AFAIK there isn't a default key source if we don't specify any.
IMU we were parsing it successfully even without any key source, which is not good and that's what we're fixing...
So it could be a breaking change for someone running a separate VC without keys (no BN, only VC without any keys, which doesn't make sense).
Running a BN node without keys is still possible in the other hand of course. If you just want to support/follow the network...